### PR TITLE
Kan 5 sathish listing vendors

### PR DESCRIPTION
**Pull request by Sathish111j**
Filtering and Search of vendor components has been created

Screenshot (44)

### CODE REVIEW

File: Frontend/src/App.jsx
Comment (aravind-digitech) [inline Frontend/src/App.jsx:1]: pls revert changes

File: Frontend/src/components/filterAndSearchVendor/Products/Product.css
Comment (aravind-digitech) [inline Frontend/src/components/filterAndSearchVendor/Products/Product.css:1]: pls rename file to product.module.css

File: Frontend/src/components/filterAndSearchVendor/Recommended/Recommended.css
Comment (aravind-digitech) [inline Frontend/src/components/filterAndSearchVendor/Recommended/Recommended.css:1]: rename

File: Frontend/src/components/filterAndSearchVendor/Recommended/Recommended.jsx
Comment (aravind-digitech) [inline Frontend/src/components/filterAndSearchVendor/Recommended/Recommended.jsx:1]: store strings in constant file like in Frontend/src/components/header/header.consts.js

File: Frontend/src/components/filterAndSearchVendor/SearchAndSort/searchAndSort.css
Comment (aravind-digitech) [inline Frontend/src/components/filterAndSearchVendor/SearchAndSort/searchAndSort.css:1]: rename

File: Frontend/src/components/filterAndSearchVendor/SearchAndSort/searchAndSort.jsx
Comment (aravind-digitech) [inline Frontend/src/components/filterAndSearchVendor/SearchAndSort/searchAndSort.jsx:1]: constants

File: Frontend/src/components/filterAndSearchVendor/Sidebar/AcceptedOrders/AcceptedOrders.jsx
Comment (aravind-digitech) [inline Frontend/src/components/filterAndSearchVendor/Sidebar/AcceptedOrders/AcceptedOrders.jsx:1]: constants

**Review by aravind-digitech — CHANGES_REQUESTED** [review]
Overall: 

1. pls look into file naming conventions
2. creating and using constants imported from another file in component jsx

File: Frontend/src/App.jsx
Comment (Sathish111j) [reply Frontend/src/App.jsx:1]: changes has been made

File: Frontend/src/components/filterAndSearchVendor/Products/Product.css
Comment (Sathish111j) [reply Frontend/src/components/filterAndSearchVendor/Products/Product.css:1]: done

File: Frontend/src/components/filterAndSearchVendor/Recommended/Recommended.css
Comment (Sathish111j) [reply Frontend/src/components/filterAndSearchVendor/Recommended/Recommended.css:1]: rename done

File: Frontend/src/components/filterAndSearchVendor/Recommended/Recommended.jsx
Comment (Sathish111j) [reply Frontend/src/components/filterAndSearchVendor/Recommended/Recommended.jsx:1]: contants have been put in other webiste

File: Frontend/src/components/filterAndSearchVendor/SearchAndSort/searchAndSort.css
Comment (Sathish111j) [reply Frontend/src/components/filterAndSearchVendor/SearchAndSort/searchAndSort.css:1]: done

File: Frontend/src/components/filterAndSearchVendor/SearchAndSort/searchAndSort.jsx
Comment (Sathish111j) [reply Frontend/src/components/filterAndSearchVendor/SearchAndSort/searchAndSort.jsx:1]: done

File: Frontend/src/components/filterAndSearchVendor/Sidebar/AcceptedOrders/AcceptedOrders.jsx
Comment (Sathish111j) [reply Frontend/src/components/filterAndSearchVendor/Sidebar/AcceptedOrders/AcceptedOrders.jsx:1]: done

**Comment by Sathish111j** [comment]
Alll required changes have been made and PR has been raised

**Comment by Sathish111j** [comment]
Alll required changes have been made and PR has been raised

**Review by aravind-digitech — APPROVED** [review]

**Review by jayakanthansubramani — APPROVED** [review]

**Pull request merged**